dolphinscheduler-dao/test/testFile.txt.crc

```diff
@@ -0,0 +1 @@
+abcdefg
```

Comment:
It's an irrelevant file, should remove it.

Reply:
ok